recipe/meta.yaml

```diff
@@ -38,11 +38,12 @@ requirements:
     - ipython
     - jinja2 >=2.1
     - jupyter_core
-    - jupyter_server >=1.4,<2
+    - jupyter_server >=1.16,<2
     - jupyterlab_server >=2.10,<3
```

Comment:
`jupyterlab_server` version `2.10` does not seem to be supported in this case.
https://github.com/jupyterlab/jupyterlab/blob/v3.4.4/setup.cfg#L38

Reply:
`~=2.10` means "_around 2.10_"

Reply:
Yes, it is equivalent to `>=2.10,==2.*`
https://peps.python.org/pep-0440/#compatible-release